### ISSUE TITLE
Questions about the reward curve and bert.

### ISSUE THREAD

**Issue by zjuAIHz** [kvablack/ddpo-pytorch#23]
Many thanks for conducting this excellent work! 

I raised 2 questions while trying to reproduce the experiments.

1. Why does the reward curve for prompt-image alignment shown in this repo fluctuate greatly, but the reward curve shown in Figure 5 in the paper is very smooth?
2. While I was experimenting, the bert scores are extremely high. For example, for the noisy image below, LLaVA generated a description: "In the image, there is a colorful, abstract, and blurry background with a mix of colors and patterns. The background is filled with a variety of colors, creating a visually interesting and dynamic scene. " The bert score of this description with the prompt "a pig riding a bike" is about 0.82. I think this may be due to the fact that the bert model I used is different from yours. So I would like to know more details about the bert used in your experiments. 

![1711874754272](https://github.com/kvablack/ddpo-pytorch/assets/100405497/b922a65c-95fd-498c-9d85-de5ad1cb2e91)

Very thanks if you can give me some help :-)

**Comment by mao-code** [comment]
Same question here

**Comment by shashankg7** [comment]
Hi, did you manage to stabilize the rewards during training?

**Comment by Lil-Shake** [comment]
Same question here. Have you solved it? Many thanks:)

**Comment by Dswn888** [comment]
Hi @zjuAIHz,

I'm also a student at ZJU, and I've encountered a similar issue while working on the prompt-image alignment experiment. Could you please share if you've managed to solve the problem, and if so, how you resolved it? Your insights would be greatly appreciated.

Thank you very much!

Best regards,
swn
